src/paperoni/__main__.py

```diff
@@ -48,52 +52,58 @@ def __call__(self, papers):
 
 
 @dataclass
-class Discover:
-    """Discover papers from various sources."""
-
+class Productor:
     command: Annotated[
         Any, FromEntryPoint("paperoni.discovery", wrap=lambda cls: Auto[cls.query])
     ]
 
+    def iterate(self):
+        for p in self.command():
+            send(discover=p)
+            yield p
```

Comment:
If I run `paperoni discover openreview --author-id "~Yoshua_Bengio1" --limit 10000`, should I see a dashboard? Is there a command line flag I have to set to see the dashboard?